tests/unit/test_opentelemetry_tracing.py

```diff
@@ -0,0 +1,104 @@
+import sys
+import pytest
+
+from importlib import reload
+import mock
+
+from google.cloud.bigquery import opentelemetry_tracing, client
+
+from opentelemetry import trace
+from opentelemetry.sdk.trace import TracerProvider
+from opentelemetry.sdk.trace.export import SimpleExportSpanProcessor
+from opentelemetry.sdk.trace.export.in_memory_span_exporter import InMemorySpanExporter
+
+TEST_SPAN_NAME = "bar"
+TEST_SPAN_ATTRIBUTES = {"foo": "bar"}
```

Comment:
```suggestion
TEST_SPAN_ATTRIBUTES = {"foo": "baz"}
```

(to avoid confusion with the `TEST_SPAN_NAME`)